egs2/qasr_tts/RESULTS.MD

```diff
@@ -0,0 +1,17 @@
+- Environments
+  - python version: `3.8.12`
+  - espnet version: `espnet 0.10.7a1`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.10.0`
+
+- Model files 
+    - training config file (teacher model): `./conf/tuning/finetune_transformer.yaml`
+    - training config file: `./conf/tuning/train_conformer_fastspeech2.yaml`
+ 
+- Results 
+  - FastSpeech2 (Trans.) w/ PWG: (R = 1) 
+      - CER: 3.9
```

Comment:
Interesting observation, can you add a note for which ASR model you are using to get the number?

Reply:
I used one of our pre-trained ASR models trained on mgb2.

Reply:
We can link to the MGB-2 recipe.